### DISCUSSION TITLE
3D Printed Stand

### DISCUSSION THREAD

**Discussion by mikeyyuen** [Phambili-Tech/Newt_Display#57] [General]
Hi I thought I'd share the stand I made in OpenSCAD for my newt and invite people to use/remix/share it.

https://github.com/mikeyyuen/Newt_3dPrint_Case

If there's a better place in the wiki or repo that it's useful to link this from let me know!

**Comment by andypiper** [comment]
Thanks for sharing this! I recently got into 3d printing myself, so I am adding this to my list of things to make 👍🏻

**Comment by justheath** [comment]
> If there's a better place in the wiki or repo that it's useful to link this from let me know!

The first place I usually look for STL is printables.com, thangs.com, and thingiverse.com. Admittedly, it took me too long to find this post with your case. Printables is also where I found spacers to use with this.
